src/Commands/Workflow/InfoCommand.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Pantheon\Terminus\Commands\Workflow;
+
+use Robo\Robo;
+
+use Pantheon\Terminus\Commands\TerminusCommand;
+use Pantheon\Terminus\Site\SiteAwareInterface;
+use Pantheon\Terminus\Site\SiteAwareTrait;
+
+class InfoCommand extends TerminusCommand implements SiteAwareInterface
+{
+    use SiteAwareTrait;
+
+    /**
+     * Show all details for a workflow including operations.
+     *
+     * @command workflow:info
+     *
+     * @param string $site_id Site name that the workflow is part of.
+     *
+     * @option string $workflow-id ID of the workflow to show.
+     * @option string $latest-with-logs Just show the latest workflow with logs.
+     *
+     * @usage terminus workflow:info <site_name> <wf-id>
+     *   Show the workflow with id <wf-id> found on site <site_name>.
+     */
+    public function info($site_id, $options = ['latest-with-logs' => false, 'workflow-id' => '']) {
+        $container = Robo::getContainer();
```

Comment:
We probably shouldn't call Robo::getContainer() here. I believe that using ContainerAwareInterface and ContainerAwareTrait would do the trick here. We'd have to do this in the TerminusCommand base class if we wanted to also create the method I suggested in my comment below.  Making every terminus command hold a reference to the container is not great.  Using a static accessor is worse.  Maybe the best thing to do would be to make a subclass of TerminusCommand that is container aware, so that commands that need the container can extend that.  I believe that Drupal Console does something like this.   The `subcommand()` function could go in that class if we did that.